src/main/kotlin/com/beust/kobalt/plugin/java/JavadocConfig.kt

```diff
@@ -0,0 +1,602 @@
+package com.beust.kobalt.plugin.java
+
+import com.beust.kobalt.api.Project
+import com.google.inject.Singleton
+import java.io.BufferedReader
+import java.io.File
+import java.io.InputStream
+import java.io.InputStreamReader
+import java.util.concurrent.TimeUnit
+
+@Singleton
+class JavadocConfig(val project: Project) {
+    val args = arrayListOf("-Xdoclint:none", "-Xmaxerrs", "1", "-quiet")
+
+    private fun removeArg(match: String, startsWith: Boolean = false, pair: Boolean = false) {
+        val it = args.iterator()
+        while (it.hasNext()) {
+            val next = it.next()
+            var removed = false
+            if (startsWith) {
+                if (next.startsWith(match)) {
+                    it.remove()
+                    removed = true
+                }
+            } else if (next == match) {
+                it.remove()
+                removed = true
+            }
+            // If it's a pair, delete the next arg too.
+            if (pair && removed && it.hasNext()) {
+                it.next()
+                it.remove()
+            }
+        }
+    }
+
+    private fun addInt(option: String, value: Int): Int {
+        args.add("-$option")
+        args.add(value.toString())
+        return value
+    }
+
+    private fun addBoolean(option: String, value: Boolean): Boolean {
+        args.remove("-$option")
+        if (value) {
+            args.add("-$option")
+        }
+        return value
+    }
+
+    private fun addString(option: String, value: String): String {
+        if (value.isNotEmpty()) {
+            args.add("-$option")
+            args.add("\"$value\"")
+        }
+        return value
+    }
+
+    private fun addStrings(option: String, vararg value: String) {
+        value.forEach {
+            addString(option, it)
+        }
+    }
+
+    private fun addOptions(option: String, first: String, second: String) {
```

Comment:
or `addPair()`. Feel free to change it, I'm not attached to it. It was refactored a few times.